packages/commonwealth/client/styles/modals/CWModalHeader.scss

```diff
@@ -0,0 +1,37 @@
+@import '../shared';
+
+.compact-modal-title {
```

Comment:
top level class name should correspond with the component name

Reply:
Addressed.